modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -148,7 +148,7 @@ def looking_for_lighthouse_claim?(claim_id:)
 
         def build_claim_structure(data:, lighthouse_id:, upstream_id:) # rubocop:disable Metrics/MethodLength
           {
-            claim_date: data[:claim_dt].present? ? data[:claim_dt].strftime('%D') : nil,
+            claim_date: data[:claim_dt].present? ? Date.parse(data[:claim_dt].to_s).strftime('%D') : nil,
```

Comment:
why did we make this change? they seem to be doing the exact same thing. 

```
(byebug) Date.parse(data[:claim_dt].to_s).strftime('%D')
"11/10/22"
(byebug) data[:claim_dt].strftime('%D')
"11/10/22"
```

Am I misinterpreting or seeing something incorrectly? Just curious.


Also, the EVSS response has a 4 digit year format. if we make a change, let's just modify it to include 4 digits for the year.

```
"phase1CompleteDate" : "09/20/2022",
"phase2CompleteDate" : "12/14/2022",
"phase6CompleteDate" : "12/14/2022",
"phaseChangeDate" : "12/14/2022",
```



Note: They will probably want this aligned with the format they're asking for ("YYYY-MM-DD") in the date ticket ([CST: Date formatting](https://vajira.max.gov/browse/API-22830)), but I can confirm with their lead and we can modify that for the other ticket. For now, let's align this to the EVSS response and just modify it to be have YYYY

Reply:
I added `.to_s`, in case the format is date, which i just saw an example failure.

Reply:
> I added `.to_s`, in case the format is date, which i just saw an example failure.

can you send me the claim for this so I can see it?

Reply:
http://localhost:3000/services/claims/v2/veterans/1012667145V762142/claims/600358586